Changes from round of JOSS review

Yurlungur opened the pull request:
<!--Provide a general summary of your changes in the title above, for
example "fix bug in ideal gas EOS.".  Please avoid
non-descriptive titles such as "Addresses issue #8576".-->

## PR Summary

This is to address comments in #427 . Thanks @guadabsb15 for the suggestions!

- Add discussion of sanity checks to getting started
- Make build/bin more consistent in documentation
- Add comment about python path
- Make sure figures display in all browsers by making png versions

<!--Please provide at least 1-2 sentences describing the pull request in
detail.  Why is this change required?  What problem does it solve?-->

<!--If it fixes an open issue, please link to the issue here.-->

## PR Checklist

<!-- Note that some of these check boxes may not apply to all pull requests -->

- [ ] Adds a test for any bugs fixed. Adds tests for new features.
- [ ] Format your changes by using the `make format` command after configuring with `cmake`.
- [ ] Document any new features, update documentation for changes made.
- [ ] Make sure the copyright notice on any files you modified is up to date.
- [ ] After creating a pull request, note it in the CHANGELOG.md file.
- [ ] LANL employees: make sure tests pass both on the github CI and on the Darwin CI

If preparing for a new release, in addition please check the following:
- [ ] Update the version in cmake.
- [ ] Move the changes in the CHANGELOG.md file under a new header for the new release, and reset the categories.
- [ ] Ensure that any `when='@main'` dependencies are updated to the release version in the package.py

junghans commented on doc/sphinx/src/getting-started.rst:
```diff
@@ -9,16 +9,35 @@ At it's most basic, you can download and compile ``singularity-eos`` with:
-  mkdir bin
-  cd bin
-  cmake -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF ..
+  mkdir -p build && cd build
+  cmake -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF -DSINGULARITY_BUILD_EXAMPLES=ON -DSINGULARITY_BUILD_TESTS=ON ..
   make -j
   make install # optional: install into directory defined via CMAKE_INSTALL_PREFIX
```
```suggestion
  cmake -B builddir -S . -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF -DSINGULARITY_BUILD_EXAMPLES=ON -DSINGULARITY_BUILD_TESTS=ON 
  cmake --build builddir --parallel
  cmake --install builddir # optional: install into directory defined via CMAKE_INSTALL_PREFIX
```
Make it more modern CMake.

Yurlungur replied:
Do you object to using `build` instead of `builddir`?

junghans replied:
No objection, my experience is just that user always confuse ``--build`` and `build`` in `cmake --build build`, one is an argument, one is a directory.

guadabsb15 commented on doc/sphinx/src/getting-started.rst:
```diff
@@ -9,16 +9,35 @@ At it's most basic, you can download and compile ``singularity-eos`` with:
-  cmake -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF ..
-  make -j
-  make install # optional: install into directory defined via CMAKE_INSTALL_PREFIX
+  cmake -B builddir -S . -DSINGULARITY_FORCE_SUBMODULE_MODE=ON -DSINGULARITY_USE_FORTRAN=OFF -DSINGULARITY_BUILD_EXAMPLES=ON -DSINGULARITY_BUILD_TESTS=ON 
```
I think that this `buildidir` should be `build`? just like the ones on line 13 and 14. I saw there was a change in name and probably that one was forgotten.

junghans replied:
Correct, my mistake!

Yurlungur commented:
Changed things back to `builddir` and did a consistency sweep through the docs. Hopefully I got them all now. I also removed the unit tests from `getting started`.
